iotdb/src/main/java/org/apache/iotdb/db/sync/receiver/ServerServiceImpl.java

```diff
@@ -0,0 +1,740 @@
+import org.apache.iotdb.service.sync.thrift.SyncService;
+import org.apache.iotdb.tsfile.file.metadata.ChunkGroupMetaData;
+import org.apache.iotdb.tsfile.file.metadata.ChunkMetaData;
+import org.apache.iotdb.tsfile.file.metadata.TsDeviceMetadata;
+import org.apache.iotdb.tsfile.file.metadata.TsDeviceMetadataIndex;
+import org.apache.iotdb.tsfile.file.metadata.enums.CompressionType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+import org.apache.iotdb.tsfile.read.ReadOnlyTsFile;
+import org.apache.iotdb.tsfile.read.TsFileSequenceReader;
+import org.apache.iotdb.tsfile.read.common.Field;
+import org.apache.iotdb.tsfile.read.common.Path;
+import org.apache.iotdb.tsfile.read.common.RowRecord;
+import org.apache.iotdb.tsfile.read.expression.QueryExpression;
+import org.apache.iotdb.tsfile.read.query.dataset.QueryDataSet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ServerServiceImpl implements SyncService.Iface {
+
+  private static final Logger logger = LoggerFactory.getLogger(ServerServiceImpl.class);
+
+  private static final FileNodeManager fileNodeManager = FileNodeManager.getInstance();
+  /**
+   * Metadata manager
+   **/
+  private static final MManager metadataManger = MManager.getInstance();
+
+  private static final String SYNC_SERVER = Constans.SYNC_SERVER;
+
+  private ThreadLocal<String> uuid = new ThreadLocal<>();
+  /**
+   * String means storage group,List means the set of new files(path) in local IoTDB and String
+   * means path of new Files
+   **/
+  private ThreadLocal<Map<String, List<String>>> fileNodeMap = new ThreadLocal<>();
+  /**
+   * Map String1 means timeseries String2 means path of new Files, long means startTime
+   **/
+  private ThreadLocal<Map<String, Map<String, Long>>> fileNodeStartTime = new ThreadLocal<>();
+  /**
+   * Map String1 means timeseries String2 means path of new Files, long means endTime
+   **/
+  private ThreadLocal<Map<String, Map<String, Long>>> fileNodeEndTime = new ThreadLocal<>();
+
+  /**
+   * Total num of files that needs to be loaded
+   */
+  private ThreadLocal<Integer> fileNum = new ThreadLocal<>();
+
+  /**
+   * IoTDB config
+   **/
+  private IoTDBConfig config = IoTDBDescriptor.getInstance().getConfig();
+
+  /**
+   * IoTDB data directory
+   **/
+  private String dataPath = config.getDataDir();
+
+  /**
+   * IoTDB  multiple bufferWrite directory
+   **/
+  private String[] bufferWritePaths = config.getBufferWriteDirs();
+
+  /**
+   * The path to store metadata file of sender
+   */
+  private ThreadLocal<String> schemaFromSenderPath = new ThreadLocal<>();
+
+  /**
+   * Sync folder path of server
+   **/
+  private String syncFolderPath;
+
+  /**
+   * Sync data path of server
+   */
+  private String syncDataPath;
+
+  /**
+   * Init threadLocal variable
+   */
+  @Override
+  public void init(String storageGroup) {
+    if (logger.isInfoEnabled()) {
```

Comment:
yes, it seems that's a little bit unnecessary, remove it.

Reply:
I think it's necessary to do preparations, because the whole synchronization process will be divided into several rounds, each round syncs one storage group.

Reply:
I mean the use of 
`if (logger.isInfoEnabled()) {`
is redundant.

Reply:
ok,fixed